src/test/java/org/springframework/data/elasticsearch/SpELEntityTests.java

```diff
@@ -47,10 +47,9 @@
 
 	@Before
 	public void init() {
-		repository.deleteAll();
```

Comment:
Please add yourself as `@author` in the class comment

Reply:
no longer needed with master as of 57ba74e5